store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/impl/RealStatefulStore.kt

```diff
@@ -0,0 +1,12 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package org.mobilenativefoundation.store.store5.impl
+
+import org.mobilenativefoundation.store.store5.Clear
+import org.mobilenativefoundation.store.store5.Read
+import org.mobilenativefoundation.store.store5.StatefulStore
+import org.mobilenativefoundation.store.store5.StatefulStoreKey
+
+internal class RealStatefulStore<Key : StatefulStoreKey, Network : Any, Output : Any, Local : Any>(
```

Comment:
what additional functionality does `RealStatefulStore` have that RealStore does not? From the look of it everything is getting delegated